core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -578,6 +579,11 @@ public SystemSessionProperties(
                         false,
                         value -> validateDoubleRange(value, FILTER_CONJUNCTION_INDEPENDENCE_FACTOR, 0.0, 1.0),
                         value -> value),
+                booleanProperty(
+                        UNESTIMATED_FILTER_COST_APPROXIMATION_ENABLED,
```

Comment:
I think the name is confusing with `default_filter_factor_enabled`. Maybe:
```
AGGREGATION_PREDICATE_DEFAULT_FILTER_FACTOR_ENABLED
```
and feature config
```
optimizer.aggregation-predicate-default-filter-factor.enabled
```

We might rename `optimizer.default-filter-factor-enabled` to something like `optimizer.global-default-filter-factor.enabled`

Reply:
I need a more generic name than "AGGREGATION_PREDICATE_..." because https://github.com/trinodb/trino/pull/11518 is going to use the same flag for inequality expression estimate.
I also want the distinction between this flag and DFF to be clear. This flag allows estimations for specific filters which cannot be accurately estimated even if full stats were present. DFF is a catch all thing which will generate estimate for any filter as long as a row count estimate exists. I'm not sure if adding `global` to the existing DFF flag will add any clarity to users as the meaning behind "global" is only going to be understood by us.
I am planning on adding a docs entry once we settle on the final name.

Reply:
`NON_ESTIMABLE_PREDICATE_APPROXIMATION_ENABLED`
or
`NON_ESTIMABLE_PREDICATE_DEFAULT_FILTER_FACTOR_ENABLED`
or
`UNSUPPORTED_PREDICATE_APPROXIMATION_ENABLED`
or
`INACCURATE_PREDICATE_APPROXIMATION_ENABLED`